src/main/java/org/folio/fqm/service/LocalizationService.java

```diff
@@ -65,6 +67,14 @@ public void localizeEntityTypeColumn(EntityType entityType, EntityTypeColumn col
       String sourceTranslation = getSourceTranslationPrefix(entityType, column.getName());
       column.setLabelAlias(sourceTranslation + column.getLabelAlias());
     }
+    if (prependShortenedName) {
+      String untranslated = ENTITY_TYPE_SHORTENED_LABEL_TRANSLATION_TEMPLATE.formatted(entityType.getName());
+      String shortenedName = translationService.format(ENTITY_TYPE_SHORTENED_LABEL_TRANSLATION_TEMPLATE.formatted(entityType.getName()));
+      if (shortenedName.equals(untranslated)) {
+        shortenedName = getEntityTypeLabel(entityType.getName());
+      }
+      column.setLabelAlias(shortenedName + " — " + column.getLabelAlias());
+    }
```

Comment:
Need at least 1 unit test for this (2 or 3 would be better, so that we get full branch coverage)